docs/self-managed/operational-guides/update-guide/850-to-860.md

```diff
@@ -35,3 +35,12 @@ This means:
 Avro schema support is the only breaking change for this Connector. If you do not use Avro schema, you can continue using the Connector as before.
 
 :::
+
+## OpenShift
+
+We added the `global.compatibility.openshift.adaptSecurityContext` variable in the values.yaml that can be used to set the following possible values:
```

Comment:
Let's add the same example we added in the [OpenShift deploy page](https://docs.camunda.io/docs/next/self-managed/setup/deploy/openshift/redhat-openshift/):

```
helm install camunda camunda/camunda-platform --skip-crds \
  --set global.compatibility.openshift.adaptSecurityContext=force
```

Reply:
done